### ISSUE TITLE
UDIS real datasets

### ISSUE THREAD

**Issue by minshu-kim** [nie-lang/UnsupervisedDeepImageStitching#51]
UDIS real datasets doesn't seem to have validation datasets.
How did you validate the model and pick the best model when fine-tuning the homography network?

**Comment by nie-lang** [comment]
It does not include a validation set. I just save the models every 100k iterations (if I remember correctly) and pick the best one in the testing set.

**Comment by minshu-kim** [comment]
Thank you for your answer
